packages/@react-types/combobox/src/index.d.ts

```diff
@@ -43,7 +46,7 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   shouldFocusWrap?: boolean
 }
 
-export interface SpectrumComboBoxProps<T> extends Omit<ComboBoxProps<T>, 'menuTrigger'>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
+export interface SpectrumComboBoxProps<T> extends Omit<AriaComboBoxProps<T>, 'menuTrigger'>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
```

Comment:
this seems like a lot to omit, I thought we were trying to include more things in the spectrum types, not exclude more, see this comment from the Issue description
> should move to Aria specific props (and also be included in the Spectrum types)